src/main/java/org/spongepowered/mod/mixin/entity/player/MixinEntityPlayerMP.java

```diff
@@ -55,7 +55,7 @@
 @NonnullByDefault
 @Mixin(EntityPlayerMP.class)
 @Implements(@Interface(iface = Player.class, prefix = "playermp$"))
-public abstract class MixinEntityPlayerMP extends EntityPlayer implements CommandSource {
+public abstract class MixinEntityPlayerMP extends SpongeUser implements CommandSource {
 
```

Comment:
This is a **mixin**, it's not a class! You cannot just arbitrarily change the parent.